October newsletter (#921)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2025-10-06-newsletter.md

- **[Policy Principles for Using GenAI in Research and Innovation](https://students.sheffield.ac.uk/research-ethics/ethics-integrity/grip/principles-using-genai-research-and-innovation)**
  The University has published Policy Principles for Using GenAI in Research and Innovation as part of the Good Research and Innovation Practices (GRIP) of the University.

- **[SciX Launches](https://scixplorer.org/scixblog/scix-launch)**
  *SciXplorer* has launched as a new platform to explore research across Earth, environmental, and space sciences.

---

## Events

### Upcoming Events in Sheffield

- **[Centre for Machine Intelligence (CMI) Town Hall](https://sheffield.ac.uk/machine-intelligence/events/cmi-town-hall)**
  *8 October, The Wave*
  Open to anyone with an interest in AI and data, featuring discussions on AI bias and updates from the CMI. Application for attending has now closed.

---

### Upcoming External Events

- **[Collaborations Workshop 2026 (CW26)](https://www.software.ac.uk/news/registration-now-open-collaborations-workshop-2026)**
  *28–30 April 2026, ICC Belfast*

>   Hybrid event on “Strengthening the Research Software Community.” Registration opens 15 September.

Reviewer: ```suggestion
  Hybrid event on “Strengthening the Research Software Community.”
```

15th Sept has already passed, registration is already open.

> - **[ReproducibiliTea Webinars](https://docs.google.com/document/d/1xuFoCPYjMzKZ2kpBVBgXHVTPXl8stghAA9e6sVWdaEA)**
>   *15–16 October 2025*
>   Two sessions exploring peer review and its diversification.
> 
> - **[7th Annual UK HPC & AI Conference](https://dirac.ac.uk/hpc-ai-advisory-council-7th-annual-uk-conference/)**
>   *14–15 October 2025, Leicester*
>   Focused on “Trustworthy Compute,” featuring talks on AI safety, sustainable infrastructure, and more.
> 
> - **[Cambridge RSE Seminar Series](https://talks.cam.ac.uk/show/index/69831)**
>   *Starts 9 October 2025 (online)*
>   A diverse series of talks covering various research software topics.

Reviewer: ```suggestion
- **[Cambridge RSE Seminar Series](https://talks.cam.ac.uk/show/index/69831)**
  *Starts 9 October 2025 (online)*
  A diverse series of talks covering various research software topics.
      - 9th October - [Chunked file formats for massive imaging datasets](https://talks.cam.ac.uk/talk/index/234385)
      - 16th October - [SerialKVM - the cheapest Software KVM](https://talks.cam.ac.uk/talk/index/234388)
      - 23rd October - [Building Reproducible Machine Learning Pipelines for inference of Galaxy Properties at Scale](https://talks.cam.ac.uk/talk/index/234391)
      - 30th October - [Portable AI software with PyTorch and Lightning](https://talks.cam.ac.uk/talk/index/234394)

- **[7th Annual UK HPC & AI Conference](https://dirac.ac.uk/hpc-ai-advisory-council-7th-annual-uk-conference/)**
  *14–15 October 2025, Leicester*
  Focused on “Trustworthy Compute,” featuring talks on AI safety, sustainable infrastructure, and more.
  
- **[ReproducibiliTea Webinars](https://docs.google.com/document/d/1xuFoCPYjMzKZ2kpBVBgXHVTPXl8stghAA9e6sVWdaEA)**
  *15–16 October 2025*
  Two sessions exploring peer review and its diversification.

```

Chronological order, also added more detail for Cambridge seminar (could reduce the number I've added)

Recognising and supporting contributions to research software is an ongoing challenge across disciplines.

- **[Code That Works Isn’t Always Code That Lasts - Software Sunstainability Institute](https://www.software.ac.uk/blog/code-works-isnt-always-code-lasts)**
  A reflection on sustainable software practices from SSI’s CW team.

- **[N8 CIR Summer 2025 Internships](https://n8cir.org.uk/news/summer-2025-internships/)** On Friday 12 September, many of our summer interns came together at The National Centre for Early Music in York to showcase their work.

- **[Migrating to Positron from VS Code](https://positron.posit.co/migrate-vscode.html)**
This document outlines how to migrate from VS Code to Positron, a data science-focused editor built on Code OSS.
- **[How to Write Docstrings in Python – Real Python](https://realpython.com/documenting-python-code/)**
Welcome to your complete guide to documenting Python code. Whether you’re documenting a small script or a large project, whether you’re a beginner or a seasoned Pythonista, this guide will cover everything you need to know.

> ### Papers

Reviewer: ```suggestion

### Papers
```

I think markdown can get upset if heading is not proceeded by a newline.

- **[IEEE CiSE Special Issue: *Research Software Engineering: Discovering and Bridging Knowledge Gaps*](https://ieeexplore.ieee.org/xpl/tocresult.jsp?isnumber=11119471&punumber=5992)**

> Special issue of IEEE journal Computing in Science & Engineering - Research Software Engineering: Discovering and Bridging Knowledge Gaps

Reviewer: ```suggestion
Special issue of IEEE journal Computing in Science & Engineering - Research Software Engineering: Discovering and Bridging Knowledge Gaps
  - Including "[Better Architecture, Better Software, Better Research](https://ieeexplore.ieee.org/document/11017520/)" (Co-authored by Robert from Sheffield RSE)
```

My paper!

Reviewer: Sarah Harris from Physics is also a co-author, if it's worth name dropping her. Other Sheff person has since left uni.

- **[Pre-print: *Hamster: A Large-Scale Study and Characterization of Developer-Written Tests*](https://arxiv.org/abs/2509.26204)**
A large-scale study revealing gaps between developer-written tests and automated test generation capabilities.

### Podcasts

- **[Bring Your Own Classroom – with The Code Refinery](https://codeforthought.buzzsprout.com/1326658/episodes/17679373-en-bring-your-own-classroom-with-the-code-refinery)**
The Code Refinery has been training researchers on how to write better software since 2016. In this episode Samantha Wittke, Richard Darst and Radovan Bast tell us how it all started, how CodeRefinery is run and what's in store for the future.

- **[Meet the 2025 SSI Fellows](https://codeforthought.buzzsprout.com/1326658/episodes/17708943-en-meet-the-2025-ssi-fellows-with-mike-simpson-esther-plomp-niko-sirmpilatze)**
Better Software - Better Research! This is the mission of the Software Sustainability Institute (SSI) since 2010. To support this, the SSI has been running a fellowship programme. In this episode you'll hear from 3 different fellows from the 2025 cohort about their motivation and plans for their fellowship.
---

## Opportunities

> - **[DisCouRSE Flexible Fund (Round 1)](https://discourse-network.github.io/funding/round-1)**

Reviewer: ```suggestion
- **[DisCouRSE Flexible Fund (Round 1)](https://discourse-network.github.io/funding/round-1)** (Closes 14th November)
```

  Funding for community-led projects to build leadership and connections across digital research technical roles. Volunteer to review applications [here](https://forms.office.com/pages/responsepage.aspx?id=_oivH5ipW0yTySEKEdmlwpofFxJMT1lCmEeJoickqdhURTFLUjE5NVc5WVdVSjI0OEk0NlY4RldFUSQlQCN0PWcu).

> - **[dRTP Skills Funding Opportunities](https://drtp-skills.ac.uk/funds/)**

Reviewer: ```suggestion
- **[dRTP Skills Funding Opportunities](https://drtp-skills.ac.uk/funds/)** (Opens *soon*)
```

  Support for Research Technical Professionals developing digital research skills.

> - **[Access to High Performance Computing Facilities (Autumn 2025)](https://www.ukri.org/opportunity/access-to-high-performance-computing-facilities-autumn-2025/)**

Reviewer: ```suggestion
- **[Access to High Performance Computing Facilities (Autumn 2025)](https://www.ukri.org/opportunity/access-to-high-performance-computing-facilities-autumn-2025/)** (Closes 6th November)
```

  Apply for computational support across the UKRI remit.

> - **[AI for Environmental Science Phase One](https://www.ukri.org/opportunity/artificial-intelligence-ai-for-environmental-science-phase-one)**

Reviewer: ```suggestion
- **[AI for Environmental Science Phase One](https://www.ukri.org/opportunity/artificial-intelligence-ai-for-environmental-science-phase-one)** (Closes 11th December)
```

  UKRI funding call for innovative uses of AI and data science in environmental research.

> - **[SSI Call for Writers and Reviewers](https://www.software.ac.uk/news/call-writers-and-reviewers-news-guides-ai-green-computing-and-edi)**

Reviewer: ```suggestion

```

Expired 3rd October

  Paid opportunities to contribute to guides on AI, Green Computing, and EDI (£600 for writing, £210 for reviewing).

> - **[SSI DisCouRSE Network+ £10k Fund](https://www.software.ac.uk/news/discourse-network-launches-ps10k-fund)**

Reviewer: ```suggestion
- **[SSI DisCouRSE Network+ £10k Fund](https://www.software.ac.uk/news/discourse-network-launches-ps10k-fund)** (Closes 14th November)
```

  Funding for projects fostering leadership and collaboration among digital research professionals.

---

## Jobs

> - **[Open Research Manager – University of Sheffield](https://jobsite.sheffield.ac.uk/job/Open-Research-Manager/1223-en_GB)** Working across the University and nationally with the UK Reproducibility Network (UKRN), you will support projects to accelerate open research practices, embed an open research culture, and enhance research transparency, reliability, and reproducibility.

Reviewer: ```suggestion
- **[Open Research Manager – University of Sheffield](https://jobsite.sheffield.ac.uk/job/Open-Research-Manager/1223-en_GB)** (Closes 22nd October)
  Working across the University and nationally with the UK Reproducibility Network (UKRN), you will support projects to accelerate open research practices, embed an open research culture, and enhance research transparency, reliability, and reproducibility. 
```

> - **[AI Research Engineer / Senior AI Research Engineer – CMI, Sheffield](https://jobsite.sheffield.ac.uk/job/AI-Research-Engineer-Senior-AI-Research-Engineer/1615-en_GB)**

Reviewer: ```suggestion
- **[AI Research Engineer / Senior AI Research Engineer – CMI, Sheffield](https://jobsite.sheffield.ac.uk/job/AI-Research-Engineer-Senior-AI-Research-Engineer/1615-en_GB)** (Closes 22nd October)
```

You will join the AI Research Engineering team within the University of Sheffield’s Centre for Machine Intelligence, supporting the UKRI-EPSRC funded UK Open Multimodal AI Network’s flagship Open Multimodal AI Benchmark (OMAIB) initiative. OMAIB develops open, community-driven datasets, evaluation protocols, baseline models, and deployment-centric tasks to accelerate multimodal AI solutions for Tomorrow’s Engineering Research Challenges.

> - **[Research Software Engineer Assistant – Durham University](https://durham.taleo.net/careersection/du_ext/jobdetail.ftl?job=25001274)**

Reviewer: ```suggestion
- **[Research Software Engineer Assistant – Durham University](https://durham.taleo.net/careersection/du_ext/jobdetail.ftl?job=25001274)** (Closes 12th October)
```

### Community

#### Digital Research Practice Support Community

The DRPS community is a group for people that support researchers in carrying out research in the digital age. Meetings
are held monthly, with discussions around events, training and opportunities related to the field.

You can join the google group
[here](https://groups.google.com/u/1/a/sheffield.ac.uk/g/digital-research-practice-support-community-group/about) to
stay informed.

> The next meeting is scheduled for 2pm on Wednesday 2025-09-17.

Reviewer: ```suggestion
The next meeting is scheduled for 2pm on Wednesday 2025-10-15.
```

@tdjames1 to confirm

Author: Oops good spot

#### LunchBytes

[LunchBytes][lunchbytes] are short talks from the research community on research
software, data, and infrastructure.

More information on future LunchBytes will be coming out over the coming months, so sign up to the [RSE mailing
list](https://groups.google.com/a/sheffield.ac.uk/g/RSE-group) if you'd like to learn more about research software and

> associated practices, or get in contact with [Farhad](mailto:farhad.allian@sheffield.ac.uk) if you would like to share

Reviewer: Not sure what's happening with this, might be worth giving him a nudge to see if this is dead in the water as it was said he'd be restarting it around now (post summer).

Author: Norbi has taken over ownership of this. I will update contact details

what you know at an upcoming session!

##### LunchBytes needs YOU

LunchBytes are organised by and for the research software community at The University of Sheffield. If you'd like to
curate a session on a topic or present something, get in touch by emailing
[lunchbytes-organisers-group@sheffield.ac.uk](mailto:lunchbytes-organisers-group@sheffield.ac.uk) or suggest topics

> [on the jamboard](https://jamboard.google.com/d/1-51cRf0pwZl8O10CnLeJGAqKcnbww-QGaYjszFK-H38/).

Reviewer: ```suggestion
```

Feels like this clashes with the above section, e.g. saying contact Farhad if you'd like to present. Who's even on this mailing list.

---
layout: post
title: October 2025 Newsletter
editor: Edwin Brown

> slug: 2025-10-01-newsletter

Reviewer 2: I think this needs to reflect the filename itself

```suggestion
slug: 2025-10-06-newsletter
```

> date: 2025-10-01T12:00:00Z+0100

Reviewer 2: Align with above date...

```suggestion
date: 2025-10-06T12:00:00Z+0100
```

news, opportunities, events and training for you.

---

## News

- **[Open Science boosts citation rates in France](https://www.ouvrirlascience.fr/for-the-first-time-on-a-scale-of-a-country-france-a-study-demonstrates-that-open-science-could-increase-the-chances-for-researchers-to-be-cited)**
*Ouvrir la Science* released a national-scale study showing that Open Science practices (like sharing preprints, data, and software) are positively correlated with citation counts.

- **[Policy Principles for Using GenAI in Research and Innovation](https://students.sheffield.ac.uk/research-ethics/ethics-integrity/grip/principles-using-genai-research-and-innovation)**
  The University has published Policy Principles for Using GenAI in Research and Innovation as part of the Good Research and Innovation Practices (GRIP) of the University.

- **[SciX Launches](https://scixplorer.org/scixblog/scix-launch)**
  *SciXplorer* has launched as a new platform to explore research across Earth, environmental, and space sciences.

---

## Events

### Upcoming Events in Sheffield

- **[Sheffield ReproducibiliTea Journal Club](https://reproducibilitea-sheffield.github.io/posts/2025-10-21/)**

>   *21st October, Room 12.1.01 (Level 1) of the Charles Street Building (Sheffield Hallam University*

Reviewer: ```suggestion
  *21st October, Room 12.1.01 (Level 1) of the Charles Street Building (Sheffield Hallam University).*
```

>   Open to anyone with an interest in AI and data, featuring discussions on AI bias and updates from the CMI. Application for attending has now closed.

Reviewer 2: Is there any point in including this if readers who haven't already registered can't sign-up?

Presumably those already registered will get notifications of the forthcoming event.

Reviewer 2: Definitely remove this now as its the 8th today.

> 

Reviewer 2: Not sure why I didn't add it to the #newswire channel but there is also a local ReproducibiliTea Journal Club coming up.

```suggestion

- **[Sheffield ReproducibiliTea Journal Club](https://reproducibilitea-sheffield.github.io/posts/2025-10-21/)**
  *21st October, Room 12.1.01 (Level 1) of the Charles Street Building (Sheffield Hallam University*
  Dr Ester Ehiyazaryan-White, Senior Lecturer in Childhood Studies will go through the paper [The Conundrum of Sharing Research Data](https://onlinelibrary.wiley.com/doi/abs/10.1002/asi.22634) and lead discussion.

```

---

### Upcoming External Events

- **[Collaborations Workshop 2026 (CW26)](https://www.software.ac.uk/news/registration-now-open-collaborations-workshop-2026)**
  *28–30 April 2026, ICC Belfast*

>   Hybrid event on “Strengthening the Research Software Community.”

Reviewer 2: Perhaps personal style but I think italics stand out better than quotes (although could maximise impact and have quotes around the italics!). Also thought it worth mentioning its from the SSI.

```suggestion
  Hybrid event on "_Strengthening the Research Software Community_" run by the Software Sustainability Institute.
```

- **[Cambridge RSE Seminar Series](https://talks.cam.ac.uk/show/index/69831)**
  *Starts 9 October 2025 (online)*
  A diverse series of talks covering various research software topics.

>       - 9th October - [Chunked file formats for massive imaging datasets](https://talks.cam.ac.uk/talk/index/234385)

Reviewer: ```suggestion
```

Too late.

      - 16th October - [SerialKVM - the cheapest Software KVM](https://talks.cam.ac.uk/talk/index/234388)
      - 23rd October - [Building Reproducible Machine Learning Pipelines for inference of Galaxy Properties at Scale](https://talks.cam.ac.uk/talk/index/234391)
      - 30th October - [Portable AI software with PyTorch and Lightning](https://talks.cam.ac.uk/talk/index/234394)

- **[7th Annual UK HPC & AI Conference](https://dirac.ac.uk/hpc-ai-advisory-council-7th-annual-uk-conference/)**
  *14–15 October 2025, Leicester*

>   Focused on “Trustworthy Compute,” featuring talks on AI safety, sustainable infrastructure, and more.

Reviewer 2: Italics and also move comma outside of quotes...

```suggestion
  Focused on “_Trustworthy Compute_”, featuring talks on AI safety, sustainable infrastructure, and more.
```

- **[ReproducibiliTea Webinars](https://docs.google.com/document/d/1xuFoCPYjMzKZ2kpBVBgXHVTPXl8stghAA9e6sVWdaEA)**
  *15–16 October 2025*
  Two sessions exploring peer review and its diversification.

- **[Bennett Institute Medicines Symposium](https://www.bennett.ox.ac.uk/events/2025-bennett-institute-medicines-symposium/)**
  *10–11 December 2025, Oxford*
  Annual symposium hosted by the Bennett Institute for Applied Data Science.

---

## Articles, Blogs, Papers & Podcasts

### Articles & Blogs

- **RSE Sheffield Blog Updates:**
  - **[Funded RSE Projects 2025](https://rse.shef.ac.uk/blog/2025-09-30-funded-proposals/)**

>     Following the success of the 2024 call for proposals the RSE team in Sheffield put a call out a second call for proposals earlier this year. The call was open to all researchers across the university and successful applicants would receive dedicated support from an RSE Team member for upto 50% FTE for a period of six months.

Reviewer 2: Just a Minute! - Repetition!

```suggestion
    Following the success of the 2024 call for proposals the RSE team in Sheffield put a second call for proposals earlier this year. This was open to all researchers across the university and successful applicants would receive dedicated support from an RSE Team member for upto 50% FTE for a period of six months.
```

- **[Telling Our Success Stories: Recognising Contributions in Research Software - Software Sunstainability Institute](https://www.software.ac.uk/blog/telling-our-success-stories-recognising-contributions-research-software)**
Recognising and supporting contributions to research software is an ongoing challenge across disciplines.

- **[Code That Works Isn’t Always Code That Lasts - Software Sunstainability Institute](https://www.software.ac.uk/blog/code-works-isnt-always-code-lasts)**
  A reflection on sustainable software practices from SSI’s CW team.

- **[N8 CIR Summer 2025 Internships](https://n8cir.org.uk/news/summer-2025-internships/)** On Friday 12 September, many of our summer interns came together at The National Centre for Early Music in York to showcase their work.

- **[Migrating to Positron from VS Code](https://positron.posit.co/migrate-vscode.html)**
This document outlines how to migrate from VS Code to Positron, a data science-focused editor built on Code OSS.

> - **[How to Write Docstrings in Python – Real Python](https://realpython.com/documenting-python-code/)**

Reviewer 2: Inconsistent spacing between list items.

Everywhere else you've used spaces which results in the items being treated as separate lists. This isn't necessarily a problem, it could be argued it makes it easier to read with more padding (I expect there is some fancy CSS that would take care of that) but here they are concurrent and this and the previous are part of the same list so it looks slightly off.
